setup.py

```diff
@@ -0,0 +1,11 @@
+from setuptools import setup
+
+setup(name='pyxdf',
+      version='0.1',
+      description='Python library for importing XDF (Extensible Data Format)',
+      url='http://github.com/sccn/xdf',
+      author='Chadwick Boulay',
+      author_email='chadwick.boulay@gmail.com',
+      license='MIT',
```

Comment:
Should be BSD.

Reply:
oh sorry I edited the branch for my project. I thought @sappelhoff has a branch that can be merged. If that is the case I will take down this pr.

Reply:
@micuat I stopped working with XDF for now and I think your branch is better suited for a merge.